packages/react/src/app-shell/AppShell.css.ts

```diff
@@ -0,0 +1,10 @@
+import { recipe } from "../vanilla-extract";
+
+export const appShell = recipe({
+  base: [
+    {
+      gap: "0",
+      h: "full",
+    },
+  ],
```

Comment:
Let's inline the sprinkles if we are not doing any complex styling or need of any selector.